### PR TITLE
added Certified

### PR DESCRIPTION
**Pull request by steven0784**
added Certified workflows

### CODE REVIEW

File: Certified/cisco_aci_example/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# **Example Cisco ACI Workflow**
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/README.md]: Change to Certified Cisco ACI Workflow

File: Certified/cisco_aci_example/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+This workflow applies to BlueCat customers who have transitioned to the Cisco ACI infrastructure for their data centers.
+
+### Requirements
+**BlueCat Gateway version:** 19.5.1 and greater <br/>
+**BAM version:** 9.1.0 and greater <br/>
+
+### Description/Example Usage
+This Certified Example Cisco ACI workflow allows you to populate Address Manager with data from your CISCO ACI environment. Using this workflow, you can import ACI Tenants and their infrastructure as well as ACI Fabric Devices into Address Manager for visibility alongside your DNS, DHCP, and IPAM infrastructure in BAM.
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/README.md]: This Certified Cisco ACI workflow...

File: Certified/cisco_aci_example/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+Before you begin, you must import the Cisco ACI workflow either by downloading it manually from GitHub, or through the GitHub Import function in the BlueCat Gateway UI. Once it is imported, you must set the necessary permissions for the workflow.
+___
+
+### Using the Cisco ACI workflow
+You can search for Cisco ACI tenants in the infrastructure by providing your Cisco APIC IP address, username, and password. As an option, you can import ACI Fabric Devices such as APIC, SPINE, and LEAF switches to the selected BAM configuration. Once you search for tenants, a table displays all tenants in the Cisco APIC. You can then select which tenants to import into Address Manager.
+
+To use the Cisco ACI workflow:
+1. Log in to BlueCat Gateway.
+2. Click **Examples > Cisco ACI**.
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/README.md]: Change Examples to Certified.

File: Certified/cisco_aci_example/README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+**Attention**
+
+Importing tenants may take a prolonged period of time depending on the number of tenants and the complexity of your infrastructure.
+___
+
+### FAQ
+
+**Do I need Cisco ACI credentials to use this workflow?**
+
+Yes. You must have ACI credentials to use this workflow. Contact your CISCO administrator to obtain credentials. You cannot search for tenants without these credentials.
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/README.md]: CISCO is in all caps in the second sentence. Change to Cisco.

File: Certified/cisco_aci_example/README.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+**Can I use an earlier version of Address Manager?**
+
+You can only use Address Manager v9.1.0 or greater. Earlier versions are not compatible with the Cisco ACI workflow.
+
+**Can I deploy changes I've made to my tenants in BAM to my ACI environment?**
+
+No. Currently, the ACI workflow only imports ACI Tenants and Fabric Devices into Address Manager, and cannot push changes back to the ACI environment.
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/README.md]: Currently, the Cisco ACI workflow...

File: Certified/cisco_aci_example/README.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+**How do I troubleshoot error messages?**
+
+Each section of the ACI workflow will display error messages relevant to that section. For example, if you have the wrong ACI credentials, you will see the appropriate error message in the UI. If you still cannot resolve the issue based on the UI error message, please refer to the log files for more detailed information.
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/README.md]: Each section of the Cisco ACI workflow...

File: Certified/cisco_aci_example/README.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+**Why can't I import ACI Fabric Devices? I'm getting an error message in the UI.**
+
+When importing ACI Fabric Devices however, you might see the following error message:
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/README.md]: Remove 'however'

File: Certified/cisco_aci_example/__init__.py [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# By: BlueCat Networks
+# Date: 2018-12-12
+# Gateway Version: 18.12.1
+# Description: Example Cisco ACI Workflows
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/__init__.py]: Change to Certified Cisco ACI Workflow

File: Certified/cisco_aci_example/__init__.py [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# By: BlueCat Networks
+# Date: 2018-12-12
+# Gateway Version: 18.12.1
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/__init__.py]: Gateway version is 19.5.1

File: Certified/cisco_aci_example/cisco_aci_example_form.py [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# By: BlueCat Networks
+# Date: 2018-12-12
+# Gateway Version: 18.12.1
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/cisco_aci_example_form.py]: 19.5.1

File: Certified/cisco_aci_example/cisco_aci_example_form.py [hunk 11]
```diff
@@ -0,0 +1,205 @@
...
+# Description: Example Gateway workflows
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/cisco_aci_example_form.py]: Certified Cisco ACI Workflow

File: Certified/cisco_aci_example/text/en.txt [hunk 12]
```diff
@@ -0,0 +1,2 @@
+title=Cisco ACI
+info=Cisco ACI Tenant Import Example Workflow.
```
Comment (delmeherbert) [inline Certified/cisco_aci_example/text/en.txt]: Change Example to Certified

File: Certified/cisco_aci_example/__init__.py (hunk 8 above)
Comment (fradvic) [reply Certified/cisco_aci_example/__init__.py]: Gateway version is incorrect, please change it to 19.5.1

Comment (delmeherbert) [reply Certified/cisco_aci_example/__init__.py]: @steven0784 version changed -- thank you. Description here still says Example Cisco ACI workflows. Please change to Certified Cisco ACI Workflow.

**Pull request merged**